flask_app/templates/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MPTCP Connection Status</title>
+    <link rel="stylesheet" type="text/css" href="/static/styles.css">
+</head>
+<body>
+    <a id="logo" href="https://mptcp.dev"><img src="/static/MPTCP_logo.svg"></a>
+    <main>
+        <h1>connection check</h1>
+        <p>
+            Multipath TCP or <a href="https://en.wikipedia.org/wiki/Multipath_TCP">MPTCP</a> is an extension to the standard <a href="https://en.wikipedia.org/wiki/Transmission_Control_Protocol">TCP</a> and is described in <a href="https://www.rfc-editor.org/rfc/rfc8684.html">RFC 8684</a>. It allows a device to make use of multiple interfaces at once to send and receive TCP packets over a single MPTCP connection. MPTCP can aggregate the bandwidth of multiple interfaces or prefer the one with the lowest latency, it also allows a fail-over if one path is down, and the traffic is seamlessly reinjected on other paths.
+        </p>
+        <p>For more informations, visit <a href="mptcp.dev">mptcp.dev</a></p>
+        <div class="container">
+            <h2>MPTCP Connection Status</h2>
+            <div id="connection-status" class="{{ state_class }}">
+                <p>You <strong>{{ state_message }}</strong> using MPTCP!</p>
+            </div>
+        </div>
+
+        <h2>Static files</h2>
+        <p>
+            You can download files of various sizes form <a href="/files/">check.mptcp.dev/files/</a>
```

Comment:
Can you use a variable instead of `check.mptcp.dev`?

Or just use `files`

```suggestion
            You can download files of various sizes form the <a href="files">files</a> directory.
```